doc/neomake.txt

```diff
@@ -56,7 +56,7 @@ fantastic vim plugins such as syntastic and dispatch.
 <
                                                              *:NeomakeCancelJob*
 :NeomakeCancelJob *{job_id}*
```

Comment:
Cool finding, I can reproduce this with `:helptags [doc_dir]`... isn't there a vim-doclint? :P